src/operator/src/insert.rs

```diff
@@ -558,7 +595,21 @@ impl Inserter {
             }
         }
 
-        Ok(table_name_to_ids)
+        let mut imme_table_ids = HashSet::new();
+        // add ttl zero regions to context
+        for table_name in table_name_to_ids.keys() {
```

Comment:
Looks like this iteration can be inlined to above loop at https://github.com/GreptimeTeam/greptimedb/blob/66c04459749703c9926b0aabfd52fccf1371bd93/src/operator/src/insert.rs#L504

Avoid to `get_table` so many times.